packages/fxa-auth-server/docs/metrics-events.md

```diff
@@ -150,7 +150,7 @@ represent a successful step
 in a sign-in or sign-up flow:
 
 | Name                                             | Description                                                                                                                                               |
-| ------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------- |
```

Comment:
Auto-format may have messed you up here.

Reply:
This one was on purpose, it just made more space for a longer description

Reply:
I think it's one column too long.

<img width="1636" alt="Screen Shot 2022-08-16 at 4 58 25 PM" src="https://user-images.githubusercontent.com/28129806/184983946-4ccf92a0-9835-49eb-98d3-94d5be49a038.png">